.eslintrc.json

```diff
@@ -1,6 +1,7 @@
 {
-  "extends": "next/core-web-vitals",
+  "extends": ["next/core-web-vitals", "plugin:@typescript-eslint/recommended"],
   "rules": {
-    "no-constant-binary-expression": "error"
+    "no-constant-binary-expression": "error",
+    "@typescript-eslint/no-explicit-any": "off"
```

Comment:
`plugin:@typescript-eslint/recommended` seems like a good addition.

`@typescript-eslint/no-explicit-any` will be fixed step by step in ongoing development.